dali/test/python/operator_1/test_numba_func.py

```diff
@@ -42,6 +46,16 @@ def set_all_values_to_255_sample(out0, in0):
     out0[:] = 255
 
 
+def set_all_values_to_1_sample_gpu(out0, in0):
+    tx, ty, tz = cuda.grid(3)
+    x_s, y_s, z_s = cuda.gridsize(3)
+
+    for z in range(tz, out0.shape[0], z_s):
+        for y in range(ty, out0.shape[1], y_s):
+            for x in range(tx, out0.shape[2], x_s):
+                out0[z][y][x] = 1
```

Comment:
Is it how it's done in numba, or can/should we use more pythonic/numpyic way?
```suggestion
       out0[tz::z_s, ty::y_s, tx::x_s] = 1
```